README.md

```diff
@@ -108,8 +122,43 @@ $ yarn install
 
 You have several rake tasks available for you:
 
-* `rake development_app`: Creates a development app inside `decidim_development` which you can use to run an application with the gems in your path.
-* `rake test_all`: Generates a test app for every engine and runs their tests.
-* `rake generate_all`: Generates all the tests apps but doesn't run the tests - this is useful is you want to run them manually afterwards.
+* `bundle exec rake development_app`: Creates a development app inside `decidim_development` which you can use to run an application with the gems in your path.
+* `bundle exec rake test_all`: Generates a test app for every engine and runs their tests.
+* `bundle exec rake generate_all`: Generates all the tests apps but doesn't run the tests - this is useful is you want to run them manually afterwards.
+* `cd <component>` and do `bundle exec rspec spec` to run those particular tests.
+
+
+### Browse Participatory Processes
+
+After you create a development app (`bundle exec rake development_app`):
+- `cd development_app`
+- `bundle exec rails s`
+- Go to 'http://localhost:3000'
+
+
+### Browse Admin Interface
+
+After you create a development app (`bundle exec rake development_app`):
+- `cd development_app`
+- `bundle exec rails s`
+- Go to 'http://localhost:3000/admin'
+- Login data: admin@decidim.org | decidim123456
+
+
+## Components
+
+
+| Component        | Description           |
+| ------------- |-------------|
+| [Admin](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-admin)      | This library adds and administration dashboard so users can manage their organization, participatory processes and all other entities. |
+| [API](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-api)      | This library exposes a GraphQL API to programatically interact with the Decidim platform via HTTP      |
+| [Comments](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-comments) | The Comments module adds the ability to include comments to any resource which can be commentable by users.      |
+| [Core](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-core) | Minimum set of funcionalities |
```

Comment:
The basics of Decidim: users, participatory processes, etc. This is the only required engine to run Decidim, all the others are optional.